SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -172,8 +169,7 @@ void USpatialSender::GainAuthorityThenAddComponent(USpatialActorChannel* Channel
 	// If this worker is EntityACL authoritative, we can directly update the component IDs to gain authority over.
 	if (StaticComponentView->HasAuthority(EntityId, SpatialConstants::ENTITY_ACL_COMPONENT_ID))
 	{
-		const FClassInfo& ActorInfo = ClassInfoManager->GetOrCreateClassInfoByClass(Channel->Actor->GetClass());
-		const WorkerAttributeSet WorkerAttribute = { ActorInfo.WorkerType.ToString() };
+		const WorkerAttributeSet WorkerAttribute = { SpatialConstants::DefaultServerWorkerType.ToString() };
```

Comment:
Nit: I think we can use SpatialConstants::UnrealServerAttributeSet or SpatialConstants::UnrealServerPermission directly in this case.